### PR TITLE
[zh-cn]sync pod-disruption-budget

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/glossary/pod-disruption-budget.md

### CODE REVIEW

File: content/zh-cn/docs/reference/glossary/pod-disruption-budget.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Pod Disruption Budget
 full-link: /zh-cn/docs/concepts/workloads/pods/disruptions/
 date: 2019-02-12
 short_description: >
- Pod Disruption Budget 是这样一种对象：它保证在主动中断（ voluntary disruptions）时，多实例应用的 {{< glossary_tooltip text="Pod" term_id="pod" >}} 不会少于一定的数量。
+ Pod Disruption Budget 是这样一种对象：它保证在主动中断（ voluntary disruptions）时，多实例应用的 Pod 不会少于一定的数量。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/glossary/pod-disruption-budget.md:7]:
```suggestion
 Pod Disruption Budget 是这样一种对象：它保证在主动中断（Voluntary Disruption）时，多实例应用的 Pod 不会少于一定的数量。
```

**Review by windsonsea — COMMENTED** [review]
Pick a nit

/approve

File: content/zh-cn/docs/reference/glossary/pod-disruption-budget.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Pod Disruption Budget
 full-link: /zh-cn/docs/concepts/workloads/pods/disruptions/
 date: 2019-02-12
 short_description: >
- Pod Disruption Budget 是这样一种对象：它保证在主动中断（ voluntary disruptions）时，多实例应用的 {{< glossary_tooltip text="Pod" term_id="pod" >}} 不会少于一定的数量。
+ Pod Disruption Budget 是这样一种对象：它保证在主动中断（Voluntary Disruption）时，多实例应用的 Pod 不会少于一定的数量。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/pod-disruption-budget.md:7]:
```suggestion
  Pod Disruption Budget 是这样一种对象：它保证在出现主动干扰（Voluntary Disruption）时，
  多实例应用的 Pod 不会少于一定的数量。
```

**Pull request merged**